bin/update-package-json.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script that will verify that all packages in our monorepo
+ * are registered in all required places and have the expected metadata in their
+ * package.json.
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const Project = require('@lerna/project');
+
+/**
+ * Check all required fields of package.json for each package on the matching
+ * with root package.json
+ * @param {Package[]} packages A list of @lerna/project packages
+ * @param {Object} rootPkg A root package.json
+ */
+async function updatePackageJsonFiles(options) {
+  const project = new Project(process.cwd());
+  const packages = await project.getPackages();
+  const rootPath = project.rootPath;
+  const rootPkg = fs.readJsonSync(path.join(rootPath, 'package.json'));
+
+  for (const p of packages) {
+    const pkgFile = path.join(p.location, 'package.json');
+    const pkg = fs.readJsonSync(pkgFile);
+    const current = JSON.stringify(pkg, null, 2);
+
+    const isTypescriptPackage = fs.existsSync(
```

Comment:
TypeScript is typically spelled as two words in CamelCase, see e.g. https://www.typescriptlang.org

```suggestion
    const isTypeScriptPackage = fs.existsSync(
```